src/main/java/pl/edu/agh/iet/katabank/Deposit.java

```diff
@@ -0,0 +1,55 @@
+package pl.edu.agh.iet.katabank;
+
+import java.math.BigDecimal;
+import java.util.Objects;
+import java.util.UUID;
+
+public class Deposit implements BankProduct {
+
+    //Komentarz do usuniecia - robie byle jaka zmiane w pliku, zeby miec z czego zrobic nowy commit i push
+    
+    private BigDecimal balance;
+    private Account connectedAccount;
+    private final UUID id;
+    private final String INCORRECT_AMOUNT_MESSAGE = "Incorrect amount to process: ";
+
+    public Deposit (Account account, BigDecimal newDepositBalance) {
```

Comment:
Maybe a better name would be initialBalance?

Reply:
Done.